oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandlerTest.java

```diff
@@ -1693,6 +1694,136 @@ void shouldErrorIfIdTokenHasIncorrectClient() throws JOSEException {
                             BASE_AUDIT_USER.withSessionId(SESSION_ID),
                             pair("description", expectedErrorObject.getDescription()));
         }
+
+        @Nested
+        class BrowserSessionIdJourney {
+
+            APIGatewayProxyResponseEvent setupWithExistingSession(
```

Comment:
Preference on private functions being defined at the bottom, after the test definitions